gcp/kbn-dev.sh

```diff
@@ -149,7 +149,6 @@ case $ACTION in
     log "Updating instance of ${TYPE} ${VALUE}, ${REPO_URL}, ${BRANCH}"
     terraform -chdir="${SCRIPT_DIR}" workspace select "${WORKSPACE_NAME}"
     PUBLIC_IP=$(terraform -chdir="${SCRIPT_DIR}" output -json | jq  -r '.public_ip.value')
-    [[ -n "$EUI" ]]; eval "ssh -q -o StrictHostKeyChecking=no ubuntu@${PUBLIC_IP} '/tmp/update_eui.sh ${EUI}'"
```

Comment:
this script is no longer necessary, if there's a `eui` or `elastic-charts` folder in `~`, `update.sh`takes care of updating these.